zcash_primitives/src/transaction/tests.rs

```diff
@@ -262,11 +264,12 @@ fn zip_0244() {
         if let Some(index) = tv.transparent_input {
             let bundle = txdata.transparent_bundle().unwrap();
             let value = bundle.authorization.input_amounts[index];
-            let script_code = &bundle.authorization.input_scripts[index];
+            let script_code = &bundle.authorization.input_scriptpubkeys[index];
             let signable_input = |hash_type| SignableInput::Transparent {
                 hash_type,
                 index,
                 script_code,
+                script_pubkey: script_code,
```

Comment:
This would be slightly clearer:
```suggestion
                script_code: script_pubkey,
                script_pubkey,
```